### PR TITLE
fix #278 Unable to scroll through left sidebar 

### LINKED ISSUE #278: Unable to scroll through left sidebar
#### Current Behavior
Unable to access nav links due to absence of scrollbar in left sidebar

image

<br><br>

Also there is this scrollbar at the bottom which needs to be fixed, happens to come only on cloud page

image

#### Expected Behavior
A scrollbar to be able to navigate through all links

image

#### Environment

- **Host OS:** Mac Linux Windows
- **Platform:** Docker or Kubernetes
- **Meshery Server Version:** stable-v
- **Meshery Client Version:** stable-v

---
#### Contributor [Guides](https://docs.meshery.io/project/contributing) and Resources
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/build-and-release)
- 📚 [Instructions for contributing to documentation](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🎨 Wireframes and [designs for Layer5 site](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Layer5 Discussion Forum](https://discuss.layer5.io) and [Layer5 Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by preyanshu**
**Notes for Reviewers**

This PR fixes #278 

**[Signed commits](../blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
-  Yes, I signed my commits.

### CODE REVIEW

**Review by Ashparshp — APPROVED** [review]
LGTM!!

**Review by iArchitSharma — CHANGES_REQUESTED** [review]
@preyanshu, first of all, thank you so much for this. However, your changes introduced a bug in the mobile view. Please fix it!
image

**Comment by preyanshu** [comment]
> @preyanshu, first of all, thank you so much for this. However, your changes introduced a bug in the mobile view. Please fix it! image

yaaa i,ll check whats wrong

**Comment by Ashparshp** [comment]
@iArchitSharma Apologies for missing that!! @preyanshu, please fix it!!

**Review by GaganpreetKaurKalsi — CHANGES_REQUESTED** [review]
Hey @preyanshu, could you fix the bottom scrollbar issue as well in this PR which was mentioned in the issue. Minor layout fix.

image

Also the scrollbar edges are getting hidden when at the extreme corners. Could you take a look at that?
image

**Comment by preyanshu** [comment]
> Hey @preyanshu, could you fix the bottom scrollbar issue as well in this PR which was mentioned in the issue. Minor layout fix.
> 
> image
> 
> Also the scrollbar edges are getting hidden when at the extreme corners. Could you take a look at that? image

so expected behaviour is that there should be no bottom scrolling bar?

**Comment by GaganpreetKaurKalsi** [comment]
> > Hey @preyanshu, could you fix the bottom scrollbar issue as well in this PR which was mentioned in the issue. Minor layout fix.
> > image
> > Also the scrollbar edges are getting hidden when at the extreme corners. Could you take a look at that? image
> 
> so expected behaviour is that there should be no bottom scrolling bar?

Yes, since the content won't be overflowing on the x-axis

**Comment by preyanshu** [comment]
hey guys @iArchitSharma @Ashparshp @GaganpreetKaurKalsi can u check this now is it okkay ?

**Comment by preyanshu** [comment]
hey guys @iArchitSharma @Ashparshp @GaganpreetKaurKalsi can u check this if its okkay?

**Review by iArchitSharma — APPROVED** [review]
Thanks LGTM now

File: assets/scss/_styles_project.scss [hunk 1]
```diff
@@ -60,10 +60,16 @@
...
+
+//main sections
 .td-outer {
   margin-top: 3rem;
 }
 
+body{
+  overflow-x: hidden;
+}
+
```
Comment (GaganpreetKaurKalsi) [inline assets/scss/_styles_project.scss:72]: Hiding the helm table contents because of this css. Unable to scroll through the whole table contents.
image

Comment (iArchitSharma) [reply assets/scss/_styles_project.scss:72]: @GaganpreetKaurKalsi thanks for pointing it out and yes please

**Pull request merged**